### PR TITLE
Demo script runnable with public node

### PR DESCRIPTION
**Pull request by danielfx90**
### What does this Pull Request do?
Adds support to run `src/demo.js` on ganache, local geth, any Ethereum public node for any net (through infura) and an RSK public node for both testnet and mainnet.

### Are there points in the code the reviewer needs to double check?
No.

### Why was this Pull Request needed?
Ease the trial of BatchPayments on different environments. Future improvements might include extend this for `src/run.js` and test cases.

### Does this Pull Request meet the acceptance criteria?
- [x] I have read the [Contribution guidelines](https://github.com/wibsonorg/BatchPayments/CONTRIBUTING.md).
- [x] I have read the [Code of Conduct](https://github.com/wibsonorg/BatchPayments/CODE_OF_CONDUCT.md).
- [x] I have read the [Security guidelines](https://github.com/wibsonorg/BatchPayments/SECURITY.md).
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.

### CODE REVIEW

File: src/demo.js [hunk 1]
```diff
@@ -132,89 +170,81 @@ async function main () {
...
+        numberOfAccounts: Math.min(accounts.length, 10),
+        bulkSize: 5,
+        numberOfPayments: 4,
+        paymentAmount: 100,
+        unlockerFee: 1,
+        collectFee: 1,
+        contracts: {
+          tokenAddress: '0x1bd7adf34f1ab0cd62bb3722b9462d30cf91c8f4',
+          batpayAddress: '0x975394d3eef7f6fe14057cb78249c8a1223c5cb4'
+        }
```
Comment (nicoayala) [inline src/demo.js]: This values should be removed

File: utils/DeploymentUtils.js [hunk 2]
```diff
@@ -29,8 +29,17 @@ exports.getEnvConfig = getEnvConfig;
...
   const config = getConfig();
-  const envConfig = getEnvConfig(environment);
-  const infura = `https://${network}.infura.io/v3/${config.infuraToken}`;
+  const envConfig = config.environments[environment] || {};
+
+  if (environment.includes('rsk')) {
+    const subdomain = network === 'testnet' ? '.testnet' : ''; 
+    publicNode = `https://public-node${subdomain}.rsk.co:443`;
+  } else {
+    publicNode = `https://${network}.infura.io/v3/${config.infuraToken}`;
```
Comment (nicoayala) [inline utils/DeploymentUtils.js]: I think that providing a publicNodeUrl in the environment config will make the project easier to integrate with other networks.

**Review by nicoayala — COMMENTED** [review]
Left some comments, let me know what do you think

File: lib/Batpay.js [hunk 3]
```diff
@@ -335,40 +354,40 @@ class BP {
...
+  }
+
   async getPaymentsLength () {
     return (await this.bp.getPaymentsLength.call())
   }
 
-  async showBalance () {
-    console.log(' '.repeat(66))
+  async showBalance(accountIds, log) {
+    log('\nFetching Accounts')
```
[the remark this answers is not in the document]
Comment (danielfx90) [reply lib/Batpay.js]: It was originally there to add an empty line to the terminal. I didn't see it. Let me change it for the `log` function. I don't have a direct replacement for `console.table`. Ideas?

File: src/demo.js (hunk 1 above)
Comment (danielfx90) [reply src/demo.js]: True

File: utils/DeploymentUtils.js (hunk 2 above)
Comment (danielfx90) [reply utils/DeploymentUtils.js]: True

**Comment by danielfx90** [comment]
@nicoayala check

**Review by nicoayala — APPROVED** [review]
🚀

File: lib/Batpay.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (nicoayala) [reply lib/Batpay.js]: None other than adding an extra package: https://www.npmjs.com/package/console.table

**Pull request merged**